src/stories/components/404/not-found-component-404.tsx

```diff
@@ -0,0 +1,308 @@
+  },
+  borderRadius: '20px',
+
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    paddingTop: '128px',
+    maxWidth: '769px',
+    margin: '0 auto',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    paddingTop: '59px',
+    maxWidth: '1130px',
+    margin: '0 auto',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    paddingTop: '59px',
+    maxWidth: '1184px',
+    margin: '0 auto',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    paddingTop: '59px',
+    maxWidth: '1184px',
+    margin: '0 auto',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    paddingTop: '59px',
+    maxWidth: '1412px',
+    margin: '0 auto',
+  },
+}));
+
+const ContainerData = styled.div({
+  zIndex: 1,
+  width: '100%',
+  height: '100%',
+  position: 'relative',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+});
+
+const LogoContainer = styled.div({
+  position: 'relative',
+  display: 'block',
+  width: '279px',
+  height: '164px',
+  marginBottom: '64px',
+  marginLeft: '32px',
+  marginRight: '32px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    marginBottom: '160px',
+    width: '580px',
+    height: '340px',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    marginBottom: '37.26px',
+    width: '675px',
+    height: '397.74px',
+  },
+});
+
+const ContainerText = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  paddingRight: '32px',
+  paddingLeft: '32px',
+  maxWidth: '343px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.between('desktop_1280', 'desktop_1440')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+    maxWidth: '100%',
+  },
+  [lightTheme.breakpoints.up('desktop_1920')]: {
+    maxWidth: '100%',
+  },
+});
+
+const TextUps = styled(Typography)<{ isLight?: boolean }>(({ isLight }) => ({
+  fontFamily: 'FT Base, sans-serif',
+  fontStyle: 'normal',
+  fontWeight: 700,
+  fontSize: '36px',
+  lineHeight: '43px',
+  letterSpacing: '0.4px',
+  color: isLight ? '#787A9B' : '#D2D4EF',
+  textAlign: 'center',
+  marginBottom: '24px',
+  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
```

Comment:
Same recommendation about the same media properties than above :)

Reply:
you are rights, i will fix it

Reply:
thanks